src/main/java/org/jenkinsci/plugins/periodicbackup/FullBackup.java

```diff
@@ -52,14 +52,15 @@
     @CheckForNull
     private final String excludesString;
 	private final File baseDir;
+	private final boolean followSymbolicLinks;
 
     public FullBackup() {
-        this(null);
+        this(null, false);
     }
 
     @DataBoundConstructor
-	public FullBackup(@CheckForNull String excludesString) {
-		this(excludesString, Jenkins.getActiveInstance().getRootDir());
+	public FullBackup(@CheckForNull String excludesString, boolean followSymbolicLinks) {
```

Comment:
If I am not mistaken the old constructor annotated with DataBoundConstructor needs to remain or old configurations will not be loaded?

Reply:
I tested with the old configuration and it is working fine.